packages/db/src/resources/bytecodes.ts

```diff
@@ -29,36 +30,9 @@ export const bytecodes: Definition<"bytecodes"> = {
     type Instruction {
       opcode: String!
       programCounter: Int!
-      meta: InstructionMeta
-      sourceRange: SourceRange
```

Comment:
I guess this is being removed due to not fitting into the structure?  I gather there is now some other mechanism for handling this?

Reply:
This is just being removed for the time being - it doesn't belong as part of `bytecodes`, but rather it makes more sense underneath `compilations` (as I said in comment above)

Reply:
Wait, if it's not associated to the bytecode, it would be associated to the contract, right?  Not the compilation?

Reply:
Yes, but SourceMap is associated to Compilation, so that's where it'd go, I think

Reply:
I'm confused.  Why would SourceMap be associated to Compilation?  I would expect each Contract to have two SourceMaps, I don't see why it would go all the way back to Compilation...

Reply:
that might be a better way to do it, but that's not where it is now :)

Reply:
Ugh.  I guess I'll approve this, since this is unrelated??  But that seems seriously messed up.  I'll need to bug you and @fainashalts about this later, I guess? :-/

Reply:
I'm still not convinced that compilation is wrong, so I anticipate your bugging :)